src/network/core.rs

```diff
@@ -36,6 +36,8 @@ impl Core {
         let mut response_net_addresses: Vec<NetAddress> = Vec::new();
         // interfaces map, but we only ever expect one, for response
         let mut interfaces: HashMap<String, types::NetInterface> = HashMap::new();
+        // is ipv6 enabled, we need to propogate this to lower stack
+        let ipv6_enabled = network.ipv6_enabled;
```

Comment:
Note: We are good here since this is a native `bool` not an `Option<bool>`.

Reply:
see my other comment, this does not work for dual stacks.

Reply:
yes added check for `dual-stack` configurations.